core-feature-pack/src/main/resources/content/bin/jboss-cli.xml

```diff
@@ -47,12 +47,27 @@
 
     <!-- Include the prompt with the command into the output for each command executed in non-interactive mode -->
     <echo-command>false</echo-command>
-    
+
     <!-- Uncomment to set the command timeout. Element value is in seconds -->
     <!-- <command-timeout>30</command-timeout> -->
 
     <!-- Uncomment to display operation responses using JSON syntax.
     By default responses are displayed using DMR string syntax. -->
     <!-- <output-json>true</output-json> -->
-    
+
+    <!-- Uncomment to display to configured CLI color output.
```

Comment:
I would say: "Uncomment to enable colouring and configure colors."

Reply:
How about `Uncomment to enable/disable coloring and configure the colors for the different outputs.`?

Reply:
Should not this be uncommented by default? It made sense to have it commented when the color feature was disabled by default, but now I think it does not have to be commented.

Reply:
Makes sense to me. The whole block?

Reply:
Yes the whole block.